lms/templates/edraak_forus/auth.html

```diff
@@ -0,0 +1,38 @@
+<%page expression_filter="h"/>
```

Comment:
Did you manage to copy this from the latest version of `student_account/login_and_register.html`??

If not, please do.

Perhaps we can add a test to ensure the files are in sync and have the minimum amount of difference. Not sure about how to do this, but it looks like a good idea.

Anyway add a comment on this file for future maintainers.